content/courses/advanced-ada/parts/resource_management/access_types.rst

```diff
@@ -733,9 +1113,13 @@ words, :ada:`A1` or :ada:`A2` allow us to access the same object in memory.
 Aliased objects
 ~~~~~~~~~~~~~~~
 
-In addition to using :ada:`new` to create aliased objects, we can indicate
-that an object is aliased by using the :ada:`aliased` keyword in the object's
-declaration: :ada:`Obj : aliased Integer;`.
+As we discussed :ref:`previously <Adv_Ada_Access_Types_Allocation>`, we use
+:ada:`new` to create aliased objects on the heap. We can also use general
+access types to access objects that were created on the stack.
```

Comment:
I like the use of "created" here. It seems clearest to me.

Reply:
Good! :-)